developer/services/qrexec.md

```diff
@@ -112,6 +113,10 @@ In the target VM, a file in either of the following locations must exist, contai
   - `/etc/qubes-rpc/RPC_ACTION_NAME` when you make it in the template qube;
   - `/usr/local/etc/qubes-rpc/RPC_ACTION_NAME` for making it only in an app qube.
 
+Files in `/run/qubes/policy.d/` are deleted when the system is rebooted.
+This is useful for temporary policy that contains the name or UUID of a disposable VM, which will not be meaningful after the system has rebooted.
+Such policy files can be created manually, but they are usually created automatically by a qrexec call to dom0.
```

Comment:
Qrexec should be capitalized as it refers to the protocol and not to program (e.g. qrexec-client-vm).